CHANGES.md

```diff
@@ -1,6 +1,6 @@
 # ChangeList
 
-## = 1.13.1 (24 April 2023) =
+## = 1.13.1 (28 April 2023) =
```

Comment:
I think this should be the date of the final release. E.g. 06 May 2023, if we decide to release it on Saturday.